.github/workflows/ci.yml

```diff
@@ -30,35 +30,44 @@ on:
 jobs:
   conan-package:
     runs-on: ubuntu-22.04
-    name: "Check Conan package"
+    container: ghcr.io/eclipse-velocitas/devcontainer-base-images/cpp:v0.2
+    name: Check Conan package
+    strategy:
+      matrix:
+        host: ["linux_x86_64"]
+      fail-fast: false
+
+
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
 
+      - name: Fix dubious ownership
+        run: |
+          git config --global --add safe.directory $( pwd )
+
       - name: install prerequisites
         run: |
           sudo apt-get update && sudo apt-get install -y ninja-build clang-format-14 clang-tidy-14 cppcheck
           pip3 install -r requirements.txt
 
-      - name: Export conan package
-        run: |
-          conan export .
-
-      - name: Check if conan package is installed
```

Comment:
why did you remove this check?

Reply:
conan create does an export to local cache and a package test, hence a conan find is not necessary